### PR TITLE
Issue 301: Add tls for reqwest

### LINKED ISSUE #301: Reqwest tls support
**Problem description**
Reqwest config should be allowed to add certificate. 

**Problem location**
clientconfig

**Suggestions for an improvement**

### PR DESCRIPTION
**Pull request by Tristan1900**
**Change log description**  
Also add cert for reqwest.

**Purpose of the change**  
Fix #301

**What the code does**  
Add cert for reqwest.

**How to verify it**

### CODE REVIEW

**Review by shrids — COMMENTED** [review]
should we add an integration test for this?

File: config/src/credentials.rs [hunk 1]
```diff
@@ -276,10 +286,14 @@ async fn send_http_request(
     payload: serde_json::Value,
     header_map: HeaderMap,
     disable_cert_verification: bool,
+    tls_certs: &[u8],
 ) -> Result<Token, reqwest::Error> {
+    let cert = Certificate::from_pem(tls_certs).expect("parse pem file");
```
Comment (claudiofahey) [inline config/src/credentials.rs:291]: Will this work if the env var pravega_client_tls_cert_path points to a file with multiple certificates or a directory with multiple certificates?

Comment (Tristan1900) [reply config/src/credentials.rs:291]: yes it should be. The tls_certs contains all the certs in a byte array format and `Certificate::from_pem` should extract mutiple certs.

**Comment by Tristan1900** [comment]
@shrids I'm not sure how to add integration test for this using what we have. I guess we will need to set up the keycloak stack to test this. What's your thought? Thanks

**Review by tkaitchuck — APPROVED** [review]

**Comment by claudiofahey** [comment]
Th below commit allows Keycloak to use the OS's certificates. I have confirmed that it works well.
https://github.com/pravega/pravega-client-rust/commit/c42d55af935d8a7bf1c3460ba2a13fc280691613

**Comment by Tristan1900** [comment]
Closing this as https://github.com/pravega/pravega-client-rust/pull/303 fixed this issue.

**Pull request closed without merging**